packages/react/src/components/pagination/pagination.tsx

```diff
@@ -129,11 +138,12 @@ export const Pagination: VoidFunctionComponent<PaginationProps> = ({
 }) => {
     const { t } = useTranslation('pagination');
     const { isMobile } = useDeviceContext();
+    const headingId = useId('heading');
```

Comment:
Ce hook a changé récemment, ce n'est plus un préfixe qu'on passe. (Je sais pu pourquoi on permettait l'override, y'a pas tant de use-case). Donc ton id va toujours être "heading" et ça marchera pas si on a deux paginations dans le même document.
```suggestion
    const headingId = useId();
```